client/components/user/docs/example.jsx

```diff
@@ -0,0 +1,22 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import UserItem from '../index';
+import user from 'lib/user';
+
+function UserItemExample() {
+	const currentUser = user().get();
```

Comment:
maybe it'll be better to user the `getCurrentUser` selector and connect `UserItemExample` to the store to reflect the current data access practices of Calypso.

Reply:
This may be a good example: https://github.com/Automattic/wp-calypso/blob/master/client/blocks/author-selector/docs/example.jsx

Reply:
Thanks @rodrigoi, I'll take a look!

Reply:
Do you think it makes more sense to move this selector to the component itself?

Reply:
Looking at other examples, the selector is typically used in a `connect` function, and it does seem to make sense to keep the data and the visual elements apart here, even though the visual element is just the `UserItem` component.

Reply:
I think it's ok to `connect()( UserItemExample )`. I don't think that `UserItem` should be a connected component.